packages/server/test/unit/config_spec.js

```diff
@@ -578,27 +570,27 @@ describe('lib/config', () => {
       })
 
       // TODO:(lachlan): after mega PR
-      context.skip('specPattern', () => {
+      context('specPattern', () => {
```

Comment:
Just want to point out that fixing this test uncovered that we aren't validating the `specPattern` property in our latest version. This would be an issue if a user had a `specPattern` in their config that wasn't either a string or an array of strings. For example, if I had 
```
specPattern: 5
```
in my configuration, I would see this error
![Screen Shot 2022-06-17 at 12 27 02 PM](https://user-images.githubusercontent.com/14275198/174388165-d7dee955-b5fc-4aa7-9489-75e371a1fc20.png)

now, I would see this error

![Screen Shot 2022-06-17 at 12 25 47 PM](https://user-images.githubusercontent.com/14275198/174388208-2965326a-989d-4887-9f25-62c74c26fa8c.png)

Reply:
Lol, as soon as we skipped a test, the functionality regressed and no-one noticed. Obvious lesson is don't skip tests!!!!! 🤦